deploy.py

```diff
@@ -19,12 +19,13 @@
 from __future__ import print_function
 
 import argparse
-import colorama
 import copy
 import os
 import shutil
 import subprocess
 import sys
+
+import colorama
```

Comment:
Out of curiosity (don't know Python), why do we need to reorder this import?

Reply:
Just a style guide thing: system imports first. And colorama is a separate package so it belongs to the 2nd set of imports.

Reply:
Oh I see. Makes sense. Thanks!